packages/koehnlab/io/basis_util.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+
+def get_multispinmat_prod(s,spat_nums,coord):
+        """
+        Returns the spin matrix of all given Spins S in the productbasis of all spins and all spatial coordinates
```

Comment:
This requires a definition of "product basis" to be useful. There are many different, equally valid product basis choices and this function returns a very specific one. Aka: define the basis in which the returned matrix will be